### PR TITLE
Use class syntax over createClass

### PR DESCRIPTION
**Pull request by kutyel**
I had to add the corresponding babel-transform-plugin but I think the result is nice and clean, had to tweak a little standard as well to use `babel-eslint` as parser as standard does not allow experimental features out-of-the-box. 😄 

---

PS: oh! I forgot to mention that I added the npm@5 lockfile as well, you can either add it or not, as you wish 😉

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +59,8 @@
     "webpack-bundle-analyzer": "~2.8.2"
   },
   "devDependencies": {
+    "babel-eslint": "^7.2.3",
```
Comment (Kikobeats) [inline package.json]: prefer ~ over ^

File: package.json [hunk 2]
```diff
@@ -60,6 +59,8 @@
...
+    "babel-plugin-transform-class-properties": "^6.24.1",
```
Comment (Kikobeats) [inline package.json]: same

File: package.json [hunk 3]
```diff
@@ -76,5 +77,8 @@
     "lint": "standard",
     "test": "echo 'not yet implemented, wanna help?'"
   },
+  "standard": {
+    "parser": "babel-eslint"
```
Comment (Kikobeats) [inline package.json]: why this?

Comment (Kikobeats) [reply package.json]: >  as parser as standard does not allow experimental features out-of-the-box

so you are removing `create-class` dependency but adding two dependencies more ¯\_(ツ)_/¯

**Review by Kikobeats — APPROVED** [review]

**Comment by Kikobeats** [comment]
I will happy to merge after resolve npm dependencies `^` 😄

Are you committing `package-lock.json` in your projects? First time I do that

**Comment by kutyel** [comment]
In the npm docs, they advice you to do so, is just like the `yarn.lock` file, lockfiles are useless if they are not part of the repo itself ¯\_(ツ)_/¯

File: package.json (hunk 3 above)
Comment (kutyel) [reply package.json]: True, for some reason, `standard` does not complain about the object-spread-operator (which is also experimental) but it does on the class-properties (stage-2). :S

Comment (kutyel) [reply package.json]: Btw, you can remove that depency by changing your `lint` script to be `"link": "standard --parser babel-eslint"` but then you have to rely on having `babel-eslint` on the global scope, just as I assume you have standard as well :)

**Comment by kutyel** [comment]
> It is highly recommended you commit the generated package lock to source control: this will allow anyone else on your team, your deployments, your CI/continuous integration, and anyone else who runs npm install in your package source to get the exact same dependency tree that you were developing on. Additionally, the diffs from these changes are human-readable and will inform you of any changes npm has made to your node_modules, so you can notice if any transitive dependencies were updated, hoisted, etc.

Taken from the [npm docs](https://docs.npmjs.com/files/package-locks)

File: package.json (hunk 3 above)
Comment (kutyel) [reply package.json]: You win haha, I don't like the `standard` tweak either, now we are even: one dependency in exchange for one devDependency 😜

File: package.json [hunk 4]
```diff
@@ -74,11 +73,8 @@
   "scripts": {
     "build": "NODE_ENV=production webpack --progress --debug --config webpack.config.js",
     "dev": "webpack-dashboard -- node bin/server",
-    "lint": "standard",
+    "lint": "standard --parser babel-eslint",
```
Comment (Kikobeats) [inline package.json:77]: babel-eslint is not necessary as dependency?

File: package.json (hunk 3 above)
Comment (Kikobeats) [reply package.json]: let's add babel-eslint as dependency, don't assume global scope

Comment (kutyel) [reply package.json]: what about `standard` then? You use it here but is not listed as devDependecies either 😕

File: package.json (hunk 4 above)
Comment (kutyel) [reply package.json:77]: Just as necessary as `standard`, only if you are going to run `npm run lint` 😉

**Comment by kutyel** [comment]
Done 😉

**Pull request merged**